pkg/deploy/util/util.go

```diff
@@ -19,6 +20,17 @@ func LatestDeploymentNameForConfig(config *deployapi.DeploymentConfig) string {
 	return fmt.Sprintf("%s-%d", config.Name, config.LatestVersion)
 }
 
+// LatestDeploymentInfo returns info about the latest deployment for a config,
+// if it exists and its current status
+func LatestDeploymentInfo(config *deployapi.DeploymentConfig, deployments *api.ReplicationControllerList) (bool, deployapi.DeploymentStatus) {
+	if config.LatestVersion == 0 || len(deployments.Items) == 0 {
+		return false, deployapi.DeploymentStatus("Unknown")
```

Comment:
I don't think it matters for where this is currently used, but I'm not sure it's a good idea to make up an enum value here. I think just returning an empty string would be better than making up an ad-hoc status since this is really supposed to be an enum type.